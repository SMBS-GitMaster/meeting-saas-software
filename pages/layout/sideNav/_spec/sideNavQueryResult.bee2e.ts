import { setupBEIntegrationTests } from '@mm/core-bloom/_spec/setupBEIntegration'

test(`a request for the data required for the side nav works`, async () => {
  const { mmGQLClient, diResolver } = await setupBEIntegrationTests()
  const { getSideNavQuery } = await import('../sideNavQuery')

  const response = await mmGQLClient.query(getSideNavQuery(diResolver), {
    queryId: 'sidenav-query',
  })

  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
  // @ts-ignore
  delete response.data.user.supportContactCode

  expect(response).toMatchSnapshot('side-nav-data')
})
