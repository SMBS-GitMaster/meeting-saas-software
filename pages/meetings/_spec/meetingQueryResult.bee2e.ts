import { setupBEIntegrationTests } from '@mm/core-bloom/_spec/setupBEIntegration'

const meetingId = '89861'

test(`a request for the data required for the meeting page works`, async () => {
  const { mmGQLClient, diResolver } = await setupBEIntegrationTests()
  const { getMeetingPageQuery } = await import('../meetingPageQuery')

  const data = await mmGQLClient.query(
    getMeetingPageQuery(diResolver)({
      meetingId,
    }),
    {
      queryId: 'meeting-page-query',
    }
  )

  expect(data).toMatchSnapshot('meeting-page-query-data')
})
