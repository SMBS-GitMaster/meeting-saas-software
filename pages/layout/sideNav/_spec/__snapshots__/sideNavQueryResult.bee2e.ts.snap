// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`a request for the data required for the side nav works: side-nav-data 1`] = `
{
  "data": {
    "user": {
      "allMeetings": NodesCollection {
        "clientSidePageInfo": {
          "lastQueriedPage": 1,
          "pageSize": 2000,
        },
        "items": [
          {
            "id": "89861",
            "type": "meeting",
          },
          {
            "id": "89876",
            "type": "meeting",
          },
        ],
        "loadingError": null,
        "loadingState": "IDLE",
        "nodes": [
          {
            "id": "89861",
            "type": "meeting",
          },
          {
            "id": "89876",
            "type": "meeting",
          },
        ],
        "onGoToNextPage": [Function],
        "onGoToPreviousPage": [Function],
        "onLoadMoreResults": [Function],
        "onPaginationRequestStateChanged": [Function],
        "pageInfoFromResults": {
          "__typename": "PageInfo",
          "endCursor": "MQ==",
          "hasNextPage": false,
          "hasPreviousPage": false,
          "startCursor": "MA==",
        },
        "pagesBeingDisplayed": [
          1,
        ],
        "totalCount": 2,
        "useServerSidePaginationFilteringSorting": true,
      },
      "allWorkspaces": NodesCollection {
        "clientSidePageInfo": {
          "lastQueriedPage": 1,
          "pageSize": 2000,
        },
        "items": [
          {
            "id": "140113",
            "type": "workspace",
          },
          {
            "id": "140113",
            "type": "workspace",
          },
        ],
        "loadingError": null,
        "loadingState": "IDLE",
        "nodes": [
          {
            "id": "140113",
            "type": "workspace",
          },
          {
            "id": "140113",
            "type": "workspace",
          },
        ],
        "onGoToNextPage": [Function],
        "onGoToPreviousPage": [Function],
        "onLoadMoreResults": [Function],
        "onPaginationRequestStateChanged": [Function],
        "pageInfoFromResults": {
          "__typename": "PageInfo",
          "endCursor": "MQ==",
          "hasNextPage": false,
          "hasPreviousPage": false,
          "startCursor": "MA==",
        },
        "pagesBeingDisplayed": [
          1,
        ],
        "totalCount": 2,
        "useServerSidePaginationFilteringSorting": true,
      },
      "avatar": null,
      "favoriteMeetings": NodesCollection {
        "clientSidePageInfo": {
          "lastQueriedPage": 1,
          "pageSize": 4,
        },
        "items": [],
        "loadingError": null,
        "loadingState": "IDLE",
        "nodes": [],
        "onGoToNextPage": [Function],
        "onGoToPreviousPage": [Function],
        "onLoadMoreResults": [Function],
        "onPaginationRequestStateChanged": [Function],
        "pageInfoFromResults": {
          "__typename": "PageInfo",
          "endCursor": null,
          "hasNextPage": false,
          "hasPreviousPage": false,
          "startCursor": null,
        },
        "pagesBeingDisplayed": [
          1,
        ],
        "totalCount": undefined,
        "useServerSidePaginationFilteringSorting": true,
      },
      "favoriteWorkspaces": NodesCollection {
        "clientSidePageInfo": {
          "lastQueriedPage": 1,
          "pageSize": 4,
        },
        "items": [],
        "loadingError": null,
        "loadingState": "IDLE",
        "nodes": [],
        "onGoToNextPage": [Function],
        "onGoToPreviousPage": [Function],
        "onLoadMoreResults": [Function],
        "onPaginationRequestStateChanged": [Function],
        "pageInfoFromResults": {
          "__typename": "PageInfo",
          "endCursor": null,
          "hasNextPage": false,
          "hasPreviousPage": false,
          "startCursor": null,
        },
        "pagesBeingDisplayed": [
          1,
        ],
        "totalCount": undefined,
        "useServerSidePaginationFilteringSorting": true,
      },
      "firstName": "Aleksandar",
      "fullName": "Aleksandar Cvijovic",
      "id": "635886",
      "lastName": "Cvijovic",
      "notifications": NodesCollection {
        "clientSidePageInfo": {
          "lastQueriedPage": 1,
          "pageSize": 2000,
        },
        "items": [],
        "loadingError": null,
        "loadingState": "IDLE",
        "nodes": [],
        "onGoToNextPage": [Function],
        "onGoToPreviousPage": [Function],
        "onLoadMoreResults": [Function],
        "onPaginationRequestStateChanged": [Function],
        "pageInfoFromResults": {
          "__typename": "PageInfo",
          "endCursor": null,
          "hasNextPage": false,
          "hasPreviousPage": false,
          "startCursor": null,
        },
        "pagesBeingDisplayed": [
          1,
        ],
        "totalCount": 0,
        "useServerSidePaginationFilteringSorting": true,
      },
      "orgSettings": {
        "businessPlanId": "89861",
        "id": "635885",
        "type": "orgSettings",
      },
      "type": "user",
    },
  },
  "error": undefined,
}
`;
